modules/apps/redirect/redirect-service/src/test/java/com/liferay/redirect/internal/util/PatternUtilTest.java

```diff
@@ -48,6 +48,16 @@ public void testAnchoredPattern() {
 			patternStrings.toString(), 1, patternStrings.size());
 	}
 
+	@Test
+	public void testCaretSlashPattern() {
+		Map<Pattern, String> patternStrings = PatternUtil.parse(
+			new String[] {"^/xyz abc"});
+
+		Assert.assertEquals("^xyz", _getFistPatternString(patternStrings));
```

Comment:
Shouldn't this method be called `_getFirstPatternString` ? (not _getFist...)

Reply:
Yes, that was a typo.